docs/01-prerequisites.md

```diff
@@ -1,8 +1,12 @@
 # Prerequisites
 
+## Red Hat Linux Variant
```

Comment:
what is this prerequisite about?

Reply:
Sorry, let me explain - whoever reads the tutorial and wants to run it, he/she must have access to a Linux box, and it has to be an RPM-based one (because you used 'systemctl status ...'). To me it looked obvious, but may be it will be less so for beginners.
Thanks, Amir

Reply:
Hi Amir, it doesn't have to be RPM-based. You can easily do this tutorial on most common Linux distros, as well as Mac OSX. You can even most of the labs (if not all) on Windows, I bet.

Reply:
So I don't think we need to include this part in the tutorial.

Reply:
OK, agree. So let propose this text, under a new section. Regarding Windows, I'm not sure whether all commands can run, so I leave that to you.

Assumptions:
This tutorial was run on Fedora 7 [or ?]. With minor modifications of some commands, it can run on any Linux and Mac OS X.

Reply:
I had Ubuntu 16.04 installed when I created this tutorial. And I'm telling you this tutorial can be completed on most Linux distros ans OSX without any modifications in commands.